src/main/jni/openat.c

```diff
@@ -9,6 +9,7 @@
 #define STR_HELPER(x) #x
 #define STR(x) STR_HELPER(x)
 
+__attribute__((noinline))
```

Comment:
if we defined `OPTIMIZE_ASM`, do we still need `noinline`?

Reply:
i wasn't able to get this to work with any other options, but i would be willing to try